srcpkgs/arduino-cli/template

```diff
@@ -1,14 +1,13 @@
 # Template file for 'arduino-cli'
 pkgname=arduino-cli
-version=0.6.0
+version=0.7.1
 revision=1
-_status="-rc1"
-wrksrc="${pkgname}-${version}${_status}"
+wrksrc="${pkgname}-${version}"
```

Comment:
this one is the default, you can remove it

Reply:
Thank you, the line should be removed